### ISSUE TITLE
A11y view is broken on dev server

### ISSUE THREAD

**Issue by zepumph** [phetsims/chipper#673]
Since we have moved dev servers from spot, this line of code is invalid:
```js
  // add brand name for dev deploys
  var brandSuffix = window.location.hostname === 'bayes.colorado.edu' ? '_phet' : '';
```

**Comment by zepumph** [comment]
This should be added into the RIAW rc branch for https://github.com/phetsims/QA/issues/125, here are the shas to cherrypick into chipper, and then to rebuild the a11y view for RIAW and check that in.

fedec69e0f821f6168f336e64f39057e7c78380f
6bddf0728e635c47d519b9b80b181f6d3eed4d40

**Comment by zepumph** [comment]
This is fixed. Closing
